packages/text-bitmap/src/formats/index.ts

```diff
@@ -1,10 +1,12 @@
 import { TextFormat } from './TextFormat';
 import { XMLFormat } from './XMLFormat';
+import { XMLTextFormat } from './XMLTextFormat';
 
 // Registered formats, maybe make this extensible in the future?
 const formats = [
     TextFormat,
     XMLFormat,
+    XMLTextFormat,
```

Comment:
I'm not super sure of this, but what about renaming `XMLTextFormat` to `XMLStringFormat` or `XMLBufferFormat`? I'd like to disambiguate from the other two formats. Right not it seems like it's a combo of Text + XML, which isn't accurate.

Reply:
yep, totally agree with you, as for me `XMLStringFormat ` will be more clear